pkg/rtc/transport.go

```diff
@@ -263,7 +263,7 @@ func (t *PCTransport) OnStreamStateChange(f func(update *sfu.StreamStateUpdate)
 	t.streamAllocator.OnStreamStateChange(f)
 }
 
-func (t *PCTransport) AddTrack(subTrack types.SubscribedTrack) {
+func (t *PCTransport) AddTrackToStreamAllocator(subTrack types.SubscribedTrack) {
```

Comment:
perhaps `AddTrack` or `AddSubscribedTrack` is more appropriate. Since it's not always adding it to the stream allocator.

Reply:
Changing it to `AddTrack/RemoveTrack` in https://github.com/livekit/livekit-server/pull/261/commits/f2b0d30b143af37f6ab00d3eb61d6eb37edc869e